ompi/mpi/c/Makefile.am

```diff
@@ -103,7 +105,6 @@ libmpi_c_mpi_la_SOURCES = \
         cart_sub.c \
         close_port.c \
         comm_accept.c \
-        comm_c2f.c \
```

Comment:
I wrote earlier on the devel ML this is a prerequisite for the second commit.
That was some time ago and i need to have an other look at this.
Withot this, the second commit cannot compile IIRC